migrations/20191105143016_build-cars-table.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("cars", table => {
+    table.increments();
+    table
+      .text("vin", 128)
```

Comment:
Vin length is between 11 and 17 characters, so there isn't a real need to set the limit to 128